packages/wonder-blocks-clickable/src/__tests__/__snapshots__/generated-snapshot.test.js.snap

```diff
@@ -20,10 +20,9 @@ exports[`wonder-blocks-clickable example 1 1`] = `
     }
   }
 >
-  <button
+  <a
```

Comment:
I'll double check why this one switched to an `a`.

Reply:
I checked the example in clickable.md which generates this snapshot and the `Clickable` doesn't have an `href` so this should be a `button`.  I'll work on fixing this.